firebase.json

```diff
@@ -2,6 +2,10 @@
   "database": {
```

Comment:
this block should also be deleted

Reply:
it's better to not delete it to ensure that database is blocked for read

Reply:
it's better to not delete it to ensure that database is blocked for read